src/neptune/new/cli/sync.py

```diff
@@ -102,6 +104,7 @@ def sync_execution(
                             container_id=container_id,
                             container_type=container_type,
                             operations=batch,
+                            upload_path=operation_storage.upload_path,
```

Comment:
If we already have such cool abstract like OperationStorage then let's use it instead of raw path. Just add method `Path OperationStorage::get_file(UploadFile op)` or something like that and use it inside `execute_operations`. Abstractions make code clean.